packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -333,20 +337,30 @@ export class ActiveWorkflowRunner {
 	 * @returns {string[]}
 	 * @memberof ActiveWorkflowRunner
 	 */
-	async getActiveWorkflows(userId?: string): Promise<IWorkflowDb[]> {
-		// TODO UM: make userId mandatory?
-		const queryBuilder = Db.collections.Workflow!.createQueryBuilder('w');
-		queryBuilder.andWhere('active = :active', { active: true });
-		queryBuilder.select('w.id');
-		if (userId) {
-			queryBuilder.innerJoin('w.shared', 'shared');
-			queryBuilder.andWhere('shared.user = :userId', { userId });
+	async getActiveWorkflows(user?: User): Promise<IWorkflowDb[]> {
```

Comment:
For now, optional `user` since `getActiveWorkflows()` is used without arg by `removeAll()`, which is used in `start.ts`, `ActiveWorkflowRunner.ts`, and `TestWebhooks.ts`. Delving there would increase the scope of the PR.